src/utils/_mixins.scss

```
@@ -0,0 +1,15 @@
+@mixin grid($columns, $screen-size, $card-size,) {
+  @media (min-width: $screen-size) {
+    grid-template-columns: repeat($columns, $card-size);
+  }
+}
+
+@mixin image-size($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin flex {
+  display: flex;
+  align-items: center;
+}
```

If mixin has name `flex` it should include only flex
```suggestion
@mixin flex {
  display: flex;
}
```